src/examples/live-data-basic/example-live-data-basic.component.ts

```diff
@@ -16,39 +18,45 @@ const EXAMPLE_TAGS = [
 })
 export class ExampleLiveDataBasicComponent implements OnDestroy {
 	static tags = EXAMPLE_TAGS;
-	title = EXAMPLE_TAGS[1];
 
+	private destroy$: Subject<void> = new Subject();
+
+	title = EXAMPLE_TAGS[1];
 	rows: Quote[];
-	timeoutId: any = null;
 
 	constructor(private dataService: DataService, private cd: ChangeDetectorRef) {
 		this.dataService.getQuotes().subscribe(quotes => {
 			this.rows = quotes;
-			this.update(true);
+			this.cd.detectChanges();
+
+			of(null).pipe(
+				takeUntil(this.destroy$),
+				switchMap(() => timer(this.random(300, 5000))),
+				repeat()
+			).subscribe(() => {
+				const arr = new Array(this.random(1, 3)).fill(0).map(() => this.random(0, this.rows.length - 1));
+				const idxSet = new Set(arr);
```

Comment:
please consider to rename to sometings like that:
arr -> rowIndices
idxSet -> uniqRowsIndices